documentation/identifiers.md

```diff
@@ -1,70 +1,80 @@
+```eval_rst
+.. _creating-identifiers:
+```
 
-You can use these existing **internal identifiers** in constructing the identifiers that you will use in your 360Giving data files. 
+## Identifier basics
 
-*However,* because there might be an overlap between the internal identifiers you use, and the internal identifiers another funder uses, in 360Giving you add a **prefix** to your internal identifiers. 
+You may already have identifiers in your own data. For example, a number for each application or grant. You can use these existing **internal identifiers** as part of your published data. 
 
-> For example, if Indigo Trust have a grant called 'Grant27', and Nominet Trust also have a grant called 'Grant27' the two will get confused when combining the two datasets. But if, when publishing, each one adds a prefix, then we end up with two unique identifiers: '360G-indigotrust-Grant27' and '360G-Nominet-Grant27'
+*However,* because there might be an overlap between the internal identifiers you use, and the internal identifiers another funder uses, you need to add a **prefix** to avoid this possible clash. 
 
-For your grants, and any other identifiers that are particular to your organisation, you use can use a **360Giving prefix**. You can get this when registering on the data registry. (For example, you might maintain your own codes to classify grants, or you might have an internal numbering scheme for organisations rather than recording charity and company numbers.)
+```eval_rst
+.. admonition:: For example
 
-For organisation identifiers, follow the [organisation identifier](organisation-identifier) guidance below which is designed to support links to be made between 360Giving, and other datasets about an organisation.
+  If Indigo Trust have a grant called 'Grant27', and Nominet Trust also have a grant called 'Grant27' the two will get confused when combining the two datasets. But if, when publishing, each one adds a prefix, then we end up with two unique identifiers: '360G-indigotrust-Grant27' and '360G-Nominet-Grant27'
+```
 
-### Using your identifiers
+For grants, and other identifiers particular to your organisation, you use can use a **360G prefix**.
 
-We use simple text identifiers in 360Giving. As well as using these identifiers in your published data, you could also tell your grantees their prefixed identifier, and encourage them to include it in any documents they publish about their funded projects, or to include it as a 'machine tag' (or hashtag) when uploading photos and videos to social media sites. 
+For organisation identifiers, if you can, use an officially recognised identifier for the organisation, following the [organisation identifier](organisation-identifier) guidance below. If you can't do this right away, you can also use your internal identifiers for grantees along with your assigned 360G prefix. 
```

Comment:
I think we need to be a more strict in our guidance. We want them to look for company and charity number, and leave the 360G prefix only for cases when we can't find identifiers. I think we should start push it now as our data quality, and not a matter of convenience to the publisher, simply because we see that working on the data as a preparation for publishing is the only time when  they will actually collect this data if it's not in their systems.

Reply:
I think its highly unlikely that grantees will start using their prefixes, and I'm not keen that we place more burden on them. The onus should be on the publisher to get this right and I agree we should be stricter in what we'll accept. Identifiers should be the string preference over a 360G prefix wherever possible.

Reply:
If you are comfortable with taking a stronger line now with publishers then happy to see this guidance become stronger. I've staged an edit then to make this just:

> For organisation identifiers, we strongly encourage you to use an officially recognised identifier for the organisation, following the [organisation identifier](organisation-identifier) guidance below.

Reply:
That looks better. The only issue is that in the guidance there is a preference for COH identifiers over CHC and we know that grantmakers collect charity numbers and have been told to acknowledge it by our board since it reduces the burden on our publishers. We would like to, as the intermediary to map those identifiers, but this is longer down the line.